### ISSUE TITLE
Show other wallet options

### ISSUE THREAD

**Issue by yknl** [hirosystems/connect#266]
Currently Connect UI only points users to download Hiro wallet if no wallet installation is detected. 

The UI should be changed to let users know of other wallet options like [Xverse](https://www.xverse.app) for mobile. Especially since Hiro wallet is not on mobile and mobile makes up more than half of all web traffic. This causes users to drop off when they visit apps.

<img width="546" alt="image" src="https://user-images.githubusercontent.com/29081231/184096922-51016a76-729f-4180-8ba8-76a6ad6c9ea6.png">

![image](https://user-images.githubusercontent.com/29081231/184097154-0cea47f0-22cc-413d-95e7-14349a613e30.png)

**Comment by markmhx** [comment]
I agree about these needs, though I'd suggest keeping the thread going on https://github.com/hirosystems/connect/issues/101 about them since we've already had some early explorations there that could be helpful for pushing this forward.

**Comment by kyranjamie** [comment]
Maybe we should co-author a SIP with a proposal for provider mgmt? @yknl

**Comment by yknl** [comment]
> I agree about these needs, though I'd suggest keeping the thread going on #101 about them since we've already had some early explorations there that could be helpful for pushing this forward.

That thread has been open for over a year. I think we should move ahead with at least a temporary solution instead of waiting for the perfect one. We'll open a pull request.

> Maybe we should co-author a SIP with a proposal for provider mgmt? @yknl
@kyranjamie what do you have in mind for the proposal?

**Comment by kyranjamie** [comment]
@yknl Perhaps this belongs in a more general SIP, but something that describes how an extension provider object is injected into a page, in a way which doesn't create naming conflicts/overwriting the same `window.StacksProvider` object.

(this happens rn with Metamask and Coinbase wallet)

**Comment by markmhx** [comment]
@yknl I presume this grant request is related? https://github.com/stacksgov/Stacks-Grant-Launchpad/issues/646

**Comment by yknl** [comment]
> @yknl Perhaps this belongs in a more general SIP, but something that describes how an extension provider object is injected into a page, in a way which doesn't create naming conflicts/overwriting the same `window.StacksProvider` object.
> 
> (this happens rn with Metamask and Coinbase wallet)

Let's start one. 


> @yknl I presume this WalletConnect grant request is related? [stacksgov/Stacks-Grant-Launchpad#646](https://github.com/stacksgov/Stacks-Grant-Launchpad/issues/646)

No, but it is related. How does stacks.js plan to be inter-operable with WalletConnect? Right now devs will have to choose one or the other.

**Comment by janniks** [comment]
https://github.com/hirosystems/connect/pull/272
